mono/utils/hazard-pointer.h

```diff
@@ -46,7 +46,7 @@ gpointer mono_get_hazardous_pointer (gpointer volatile *pp, MonoThreadHazardPoin
 
 
 void mono_thread_small_id_free (int id);
-int mono_thread_small_id_alloc (void);
+MONO_API int mono_thread_small_id_alloc (void);
```

Comment:
It is. It's used in `init_thread` in case the thread hasn't attached yet, in which case `mono_thread_info_get_small_id` returns `-1`.

I think you might've forgotten to click "load diff" on `mono-profiler-log.c` in the full changes view. Great GitHub feature, right?